### PR TITLE
Fixing 3648 - now non updated references are shown only when non zero

### PR DESCRIPTION
**Pull request by diego898**
This is a quick fix for #3648 

Signed-off-by: Diego Mesa <diego898@gmail.com>

### CODE REVIEW

File: core/ui/EditTemplate/title.tid [hunk 1]
```diff
@@ -35,6 +35,10 @@ tags: $:/tags/EditTemplate
 
 <$checkbox tiddler="$:/config/RelinkOnRename" field="text" checked="yes" unchecked="no" default="no"> {{$:/language/EditTemplate/Title/Relink/Prompt}}</$checkbox>
 
+<$set name="backlinks" filter="[title<fromTitle>backlinks[]]">
+
+<$reveal type="nomatch" default=<<backlinks>> text="">
+
```
Comment (pmario) [inline core/ui/EditTemplate/title.tid]: Please also check the DOM. There are several paragraph elements, that shouldn't be there. Some text nodes that contain no text and so on.

Comment (diego898) [reply core/ui/EditTemplate/title.tid]: hey @pmario, Im not exactly familiar with that - I just tried to follow the convention of the file - I guess that resulted in some extra empty elements. Its never really clear to me when a new line will show up or not. Any help you can provide would be greatly appreciated!

Comment (diego898) [reply core/ui/EditTemplate/title.tid]: Hey Mario - just a bump on this. I'm eager to try and contribute something to TW!

**Comment by diego898** [comment]
just a bump on asking for advice!

**Comment by Jermolene** [comment]
Hi @diego898 the line spacing you've got looks consistent with the rest of the tiddler. However, I think it may be better to fix this with a `<$list>` widget to avoid introducting the DOM element that the `<$reveal>` widget always creates.

**Comment by diego898** [comment]
Hey @Jermolene - Ive updated the PR to use the list widget. Thank you for the help!

**Comment by diego898** [comment]
A slight update to make sure I dont overwrite the `<<currentTiddler>>` variable

**Comment by diego898** [comment]
@Jermolene could this make it into 5.1.20? If not, what else is necessary?

**Comment by Jermolene** [comment]
Thanks @diego898 looks good.

**Pull request merged**